ruby/vampires.rb

```diff
@@ -1,5 +1,7 @@
 require 'date'
 
+#this line added to submit PR for teacher review
+
```

Comment:
Nice work!  On line 87 consider this


I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.

```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```